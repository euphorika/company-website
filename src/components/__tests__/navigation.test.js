import React from "react"
import renderer from "react-test-renderer"

import Navigation from "../navigation"

describe("Navigation", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Navigation fontColor="inherit" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
