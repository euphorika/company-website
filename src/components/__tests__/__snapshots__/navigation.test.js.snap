// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders correctly 1`] = `
<div
  className="pageNavigation"
>
  <nav>
    <ul>
      <li
        className="entry"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Euphorika
        </a>
      </li>
      <li
        className="entry"
      >
        <a
          href="/service/"
          onClick={[Function]}
        >
          Leistungen
        </a>
      </li>
      <li
        className="entry"
      >
        <a
          href="/best-cases/"
          onClick={[Function]}
        >
          Best Cases
        </a>
      </li>
      <li
        className="entry"
      >
        <a
          href="/jobs/"
          onClick={[Function]}
        >
          Jobs
        </a>
      </li>
      <li
        className="entry"
      >
        <a
          href="/contact/"
          onClick={[Function]}
        >
          Kontakt
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="closeNavigation"
    onClick={[Function]}
    role="button"
  >
    <svg
      viewBox="0 0 172.26 174.13"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M172.26 28.71L143.55 0 86.13 57.42 28.71 0 0 28.71l57.42 57.42L0 143.54l28.71 28.72 57.42-57.42 57.42 57.42 28.7-28.72-57.41-57.41z"
        fill="inherit"
      />
    </svg>
  </div>
</div>
`;
